tests/testtemplate/template_cleantest_test.go

```diff
@@ -1,12 +1,13 @@
-package e2e
+package testtemplate
```

Comment:
we can call it just as `template` it is already in `tests` dir, so it is clear what tempalte is for

Reply:
fixed